### DISCUSSION TITLE
how to capture certain special tips?

### DISCUSSION THREAD

**Discussion by skyheros001** [Rapptz/discord.py#9526] [Q&A]
Currently, I can only send messages and monitor messages. I'm not sure how to capture certain special tips, such as when using Midjourney in Discord and receiving a message from the Midjourney bot saying: 'Banned prompt detected. Sorry! Our AI moderator thinks this prompt is probably against our community standards.' My question is, how can I use this project to properly capture these special bot tips?

**Comment by SebbyLaw** [comment]
You don't
